jasmine/index.html

```diff
@@ -0,0 +1,305 @@
+<!doctype html>
+<html lang="en">
+
+	<head>
+		<meta charset="utf-8">
+
+		<title>JS401 Intro to Unit Testing with Jasmine</title>
+
+		<meta name="description" content="This workshop will provide an introduction to Test Driven Development, specifically in the Behavior Driven Development style, using Jasmine.">
+		<meta name="author" content="Girl Develop It San Francisco">
+
+		<meta name="apple-mobile-web-app-capable" content="yes" />
+		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+		<link rel="stylesheet" href="css/reveal.css">
+		<link rel="stylesheet" href="css/theme/gdilight.css" id="theme">
+
+		<!-- For syntax highlighting -->
+		<!-- light editor --><link rel="stylesheet" href="lib/css/light.css">
+		 <!-- dark editor<link rel="stylesheet" href="lib/css/dark.css"> -->
+
+		<!-- If use the PDF print sheet so students can print slides-->
+
+		<link rel="stylesheet" href="css/print/pdf.css" type="text/css" media="print">
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+		<!--[if lt IE 9]>
+		<script src="lib/js/html5shiv.js"></script>
+		<![endif]-->
+	</head>
+
+	<body>
+
+		<div class="reveal">
+
+			<!-- Any section element inside of this container is displayed as a slide -->
+			<div class="slides">
+
+		        <!-- Opening -->
+				<section>
+					<h1>JS401</h1>
+					<h3>Intro to Unit Testing with Jasmine</h3>
+					<p>
+						<small>Slides: <a href ="http://www.teaching-materials.org/jasmine">http://www.teaching-materials.org/jasmine</a></small>
+					</p>
+				</section>
+
+		        <!--  High level overview of testing, TDD, and BDD -->
+				<section>
+					<h2>Why test?</h2>
+				</section>
+				<section>
+					<h3>Code is for humans</h3>
+					<h4>(and so are the bugs)</h4>
+					<ul>
+						<li>set expectations</li>
+						<li>think through the steps</li>
+						<li>break down the pieces</li>
+						<li>ensure ease of upgrades and deprecations</li>
+					</ul>
+				</section>
+				<section>
+					<h3>Benefits of testing</h3>
+					<ul>
+						<li>shorten feedback loop</li>
+						<li>prevent regressions</li>
+						<li>document code</li>
+					</ul>
+				</section>
+				<section>
+					<h3>What is TDD?</h3>
+					<br>
+					<p><strong>Test-driven Development (TDD)</strong> is an approach to programming.</p>
+					<br>
+					<ol>
+						<li>Write an automated (failing) test. This test will describe the desired behavior
+						<li>Write a small amount of code that will make the test pass</li>
+						<li>Repeat</li>
```

Comment:
Do you think it'd help to compare that to the style of programming where we write code first and then write tests? not sure if theres a name for that.